tests/test_urls.py

```diff
@@ -61,11 +61,6 @@ def test_activity_page_401_if_not_logged_in(self):
         rv = self.app.get('/me')
         self.assertEqual(rv.status_code, 401)
 
-    def test_activity_page_403_view_other_user_activity(self):
```

Comment:
Hello Mike!

I wrote the above test for error code 403. When I ran nosetests, I realized
that the test would assert for 401(page not found) and not 403. There was
already a test for 401.

Hence I removed this test.

Regards,
Deepthi
On Mar 23, 2016 10:18 PM, "Mike Taylor" <notifications@github.com> wrote:

> In tests/test_urls.py
> <https://github.com/webcompat/webcompat.com/pull/958#discussion_r57189196>
> :
>
> > @@ -61,11 +61,6 @@ def test_activity_page_401_if_not_logged_in(self):
> >          rv = self.app.get('/me')
> >          self.assertEqual(rv.status_code, 401)
> >
> > -    def test_activity_page_403_view_other_user_activity(self):
>
> What's the motivation for removing this test?
>
> (Sorry, I'm only slightly up to speed on the evolution of this issue,
> trying to reboot my brain)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/webcompat/webcompat.com/pull/958/files/c844bede9c8945e5848f2f0d4d46179e392b969e..cab3a290bcc80b2354827f760a9389d950399aa5#r57189196>
>